lib/astronoby/aberration.rb

```diff
@@ -22,27 +22,22 @@ def apply
       delta_longitude = Astronoby::Angle.as_degrees(
         -20.5 *
         Math.cos(
-          @sun_longitude.to_radians.value - @coordinates.longitude.to_radians.value
-        ) / Math.cos(@coordinates.latitude.to_radians.value) / 3600
+          @sun_longitude.radians - @coordinates.longitude.radians
+        ) / Math.cos(@coordinates.latitude.radians) / 3600
       )
 
       delta_latitude = Astronoby::Angle.as_degrees(
         -20.5 *
-        Math.sin(
-          @sun_longitude.to_radians.value -
-          @coordinates.longitude.to_radians.value
-        ) *
-        Math.sin(@coordinates.latitude.to_radians.value) / 3600
+        Math.sin(@sun_longitude.radians - @coordinates.longitude.radians) *
+        Math.sin(@coordinates.latitude.radians) / 3600
```

Comment:
Similarly here, if you had `Angle#sin` and `Angle#-` implemented this method would look like:

```ruby
(@sun_longitude - @coordinates.logitude).sin * @coordinates.lattitude.sin / 3600
```

Note that you don't need to implement `Angle#*` or `Angle#/` because `Angle#sin` would return a float, not an angle.